src/cmd-build

```diff
@@ -283,7 +283,7 @@ for itype in "${IMAGE_TYPES[@]}"; do
               /usr/lib/coreos-assembler/gf-oemid "$(pwd)"/"${img_base}" "$(pwd)"/"${img_qemu}" qemu
               # Clear the MCS SELinux labels
               # See https://github.com/coreos/coreos-assembler/issues/292
-              chcon -vl s0 "${img_qemu}"
+              chcon -vl s0 "${img_qemu}" || echo "chcon failed. This is expected if SELinux is not enabled"
```

Comment:
How about something like:

```sh
if [[ $(getenforce || :) == Enforcing ]]; then
  chcon ...
fi
```

?

That way, we make sure we still fail if SELinux is enabled and we're missing perms or something.